app/experimenter/experiments/models/nimbus.py

```diff
@@ -458,53 +457,48 @@ def __str__(self):  # pragma: no cover
 
 
 class NimbusChangeLogManager(models.Manager):
+    def latest_change(self):
+        return self.all().order_by("-changed_on").first()
+
     def latest_review_request(self):
-        return (
-            self.all()
-            .filter(
-                Q(old_status=NimbusExperiment.Status.DRAFT)
-                | Q(old_status=NimbusExperiment.Status.PREVIEW),
-            )
-            .filter(
-                old_publish_status=NimbusExperiment.PublishStatus.IDLE,
-                new_status=NimbusExperiment.Status.DRAFT,
-                new_publish_status=NimbusExperiment.PublishStatus.REVIEW,
+        change = self.latest_change()
```

Comment:
Oh technically this could return `None` so in theory you'd wanna check if it's truthy before accessing it fields.  In theory, we never hit this case cuz as soon as an experiment is created it has the creation change, so it's probably not a big deal, just wanted to point it out in case you see something wonky in testing.